### PR TITLE
Adds the resistor color exercise

### PR DESCRIPTION
**Pull request by camilopayan**
Not sure how to run the tests for my example code file. In other language tracks I saw there was documentation for it, but didn't see that here.

### CODE REVIEW

**Comment by camilopayan** [comment]
Also, for what it's worth, I was aiming for this to count toward Hacktoberfest 2020, so lmk if you plan to add Hacktoberfest as a topic to the repo.

File: exercises/resistor-color/example.go [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+	"regexp"
+	"strings"
+)
+
+type Code struct {
+	name string
+	code int
+}
+
+var colors = Code{
```
Comment (tehsphinx) [inline exercises/resistor-color/example.go:13]: I think this is supposed to be:
```suggestion
var colors = []Code{
```

File: exercises/resistor-color/resistor_color.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
+// This is a "stub" file.  It's a little start on your solution.
```
Comment (tehsphinx) [inline exercises/resistor-color/resistor_color.go:1]: I'd remove all the stub comments. This is a later exercise, so students know by now where to put code.

**Comment by tehsphinx** [comment]
Probably someone else who knows the structure of exercises better than me should look over this. `example.log` can't be in the same folder as the solution / stub-file -- the function names would clash.

**Comment by jmrunkle** [comment]
This PR has not been touched in at least 6 months. If this is still being worked on, please respond and we can continue the review process (you will probably have to sync / rebase). Otherwise I intend to close this pull request in a week.

**Pull request closed without merging**